docs/examples/guided-set-speed-yaw-demo.rst

```diff
@@ -27,28 +27,48 @@ The example is :ref:`documented in the source code <guided_example_source_code>`
 Running the example
 ===================
 
-The vehicle and DroneKit should be set up as described in :ref:`get-started`.
+The example can be run as described in :doc:`running_examples` (which in turn assumes that the vehicle
+and DroneKit have been set up as described in :ref:`get-started`).
+
 If you're using a simulated vehicle, remember to :ref:`disable arming checks <disable-arming-checks>` so 
 that the example can run.
 
-Once *MAVProxy* is running and the API is loaded, you can start the example by typing: ``api start guided_set_speed_yaw.py``.
+In summary, after cloning the repository:
 
-.. note:: 
+#. Navigate to the example folder as shown:
+
+   .. code-block:: bash
+
+       cd dronekit-python\examples\guided_set_speed_yaw\
```

Comment:
To be compatible, probably use forward slashes `/` since this will work on Windows and Linux alike